vortex-file/src/strategy.rs

```diff
@@ -63,15 +71,30 @@ impl WriteStrategyBuilder {
         self
     }
 
+    /// Override the minimum bytesize of a block for compression.
+    pub fn with_compression_block_min_bytesize(mut self, min_bytesize: Option<u64>) -> Self {
```

Comment:
a bit wordy? maybe something like

```suggestion
    pub fn with_compression_block_size(mut self, block_size: u64) -> Self {
```

Also I think this should take u64 instead of Option<u64>